exercises/concept/plane-tickets/.docs/introduction.md

```diff
@@ -0,0 +1,96 @@
+# Instructions
+
+A generator in Python is a _callable function_ that returns a [lazy iterator](https://en.wikipedia.org/wiki/Lazy_evaluation).
+
+_Lazy iterators_ are similar to `lists`, and other `iterators`, but with one key difference: They do not store their `values` in memory, but _generate_ their values when needed.
+
+## Constructing a generator
+
+Constructing a `generator` is a bit different than a normal `function`. You will need the `yield` expression, which we will go into depth with [later](#the-yield-expression).
+
+Lets say you want to construct a `generator` that generates all the _squares_ from a list of numbers. You would construct that function like this:
+
+```python
+>>> def squares(list_of_numbers):
+>>>     for number in list_of_numbers:
+>>>         yield number ** 2
+```
+
+## Using a generator
+
+It is possible to use any Python _function_ or _object_ that requires an `iterator` as an argument.
```

Comment:
```suggestion
Generators can usually be used in place of most `iterables` in Python. This includes  _functions_ or _objects_ that require an `iterable`/`iterator` as an argument.
```

Reply:
It might also be worth calling out explicitly that there's no `len()`

Reply:
I added most of this!